include/nbl/builtin/hlsl/bxdf/common.hlsl

```diff
@@ -163,7 +172,10 @@ struct SIsotropic
 
     RayDirInfo getV() NBL_CONST_MEMBER_FUNC { return V; }
     vector3_type getN() NBL_CONST_MEMBER_FUNC { return N; }
-    scalar_type getNdotV() NBL_CONST_MEMBER_FUNC { return NdotV; }
+    scalar_type getNdotV(BxDFClampMode _clamp = BxDFClampMode::BCM_NONE) NBL_CONST_MEMBER_FUNC
+    {
+        return hlsl::mix(math::conditionalAbsOrMax<scalar_type>(_clamp == BxDFClampMode::BCM_ABS, NdotV, 0.0), NdotV, _clamp == BxDFClampMode::BCM_NONE);
```

Comment:
don't you want to make a `bxdf::clamp(x,mode)` for this so you don't need two `mix` everywhere?